backend/tests/apps/mentorship/model/test_mentee_module.py

```
@@ -0,0 +1,71 @@
+
+
+class TestMenteeModule:
+    def setup_method(self):
+        self.github_user = MagicMock(spec=GithubUser, login="testmentee")
+        self.mentee = MagicMock(spec=Mentee, github_user=self.github_user, __str__=lambda self: "testmentee")
+        self.program = MagicMock(spec=Program, name="Test Program", key="test-program")
+        self.module = MagicMock(spec=Module, name="Test Module", key="test-module", program=self.program, __str__=lambda self: "Test Module")
+
+    @patch("apps.mentorship.models.MenteeModule.objects.create")
+    def test_create_mentee_module(self, mock_create):
+        start_time = datetime(2023, 1, 1, 10, 0, 0)
```

Ruff (0.14.10)

`datetime.datetime()` called without a `tzinfo` argument

(DTZ001)